docs/guides/01-getting-started/prerequisites.md

```diff
@@ -0,0 +1,214 @@
+---
+sidebar_position: 1
+pagination_next: guides/getting-started/beginning-tutorial
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Prerequisites
+
+:::tip Quick Overview
+Check out [Tauri in 100 Seconds](https://www.youtube.com/watch?v=-X8evddpu7M) by [Fireship](https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA) on YouTube for a quick introduction to Tauri
+
+<div className="videowrapper">
+  <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/-X8evddpu7M" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
+:::
```

Comment:
Do you think this makes the most sense here? 🤔 I would have put it on the main page below the feature cards maybe

Reply:
I don't recommend putting it on index since it isn't our content. Once we have guides built out, I want to make a quick "landing" page when the user clicks "Guides" in the navbar that leads to the respective sections. I think moving the video to that page once we have it built out would be better. But having it here for now at least gives it visibility

Reply:
Hmm yeah alright 🤔